interface/orders/orders_results.php

```diff
@@ -766,20 +766,20 @@ function validate(f) {
     if ($reviewauth) {
         ?>
      <center><p>
-         <input type='submit' name='form_submit' value='<?php xl('Sign Results', 'e'); ?>' />
+         <input type='submit' name='form_submit' value='<?php xlt('Sign Results', 'e'); ?>' />
      </p></center>
     <?php
     } else {
         ?>
      <center><p>
-         <input type='button' name='form_submit' value='<?php xl('Sign Results', 'e'); ?>' onclick="alert('<?php xl('Not authorized', 'e') ?>');" />
+         <input type='button' name='form_submit' value='<?php xlt('Sign Results', 'e'); ?>' onclick="alert('<?php xlt('Not authorized', 'e') ?>');" />
```

Comment:
The `Sign Results` is within an html attribute, so use xla. Also need to remove the `e` parameter and add the `echo`

And the other one is an odd case, where javascript context is embedded in html context, so we actually use a function that will escape both (if this one makes your head hurt, don't worry about it; it takes a while to figure it out). Here is what the above line will then look like:
```
<input type='button' name='form_submit' value='<?php echo xla('Sign Results'); ?>' onclick="alert(<?php echo attr_js(xl('Not authorized')) ?>);" />
```

Reply:
Okay, interesting!  I'll have to review this a few times to understand it - thank you for the explanation!